src/main/java/com/pnudev/springonlinedelivery/Menu_Item.java

```diff
@@ -0,0 +1,24 @@
+package com.pnudev.springonlinedelivery;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Table
+@Entity
+public class Menu_Item {
+	@Id
+	public long id;
+	public String name;
+	public String summary;
+	public double size;
+	public double price;
+	
+	Menu_Item(long id, String name, String summary, double size, double price){
```

Comment:
Use Lombok to automatically generate getters, setters and constructors

Reply:
Lombok implemented